### PR TITLE
style(NcHeaderMenu): pass text-on-background color for header menu

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

- Fix https://github.com/nextcloud/notifications/issues/1962

### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
image | image

### 🚧 Tasks

- [ ] Drop internal styles at server:
  - [ ] UnifiedSearch
  - [ ] ContactsMenu
  - [ ] AppMenu (NcActions `aria-label="More apps"`)

### 🏁 Checklist

- [x] ⛑️ Tests are included or are not applicable
- [x] 📘 Component documentation has been extended, updated or is not applicable
- [ ] 3️⃣ Backport to `next` requested with a Vue 3 upgrade

### CODE REVIEW

File: src/components/NcHeaderMenu/NcHeaderMenu.vue [hunk 1]
```diff
@@ -337,7 +337,7 @@ $externalMargin: 8px;
 
 		// header is filled with primary or image background
 		filter: none !important;
-		color: var(--color-primary-text) !important;
+		color: var(--color-background-plain-text) !important;
```
Comment (susnux) [inline src/components/NcHeaderMenu/NcHeaderMenu.vue:340]:
```suggestion
		color: var(--color-background-plain-text, var(--color-primary-text)) !important;
```

**Review by susnux — APPROVED** [review]

**Review by DorraJaouad — APPROVED** [review]

**Comment by nickvergessen** [comment]
"More apps" … is still white? :D

**Comment by Antreesy** [comment]
> is still white?

Should be fixed in server repo, as I see it

**Pull request merged**